src/t2iapi/biceps/instanceidentifier.proto

```diff
@@ -0,0 +1,39 @@
+/*
+This Source Code Form is subject to the terms of the MIT License.
+Copyright (c) 2024 Draegerwerk AG & Co. KGaA.
+
+SPDX-License-Identifier: MIT
+*/
+
+syntax = "proto3";
+
+package t2iapi.biceps;
+
+option java_package = "com.draeger.medical.t2iapi.biceps";
+option java_outer_classname = "InstanceIdentifierProto";
+
+import "t2iapi/biceps/codedvalue.proto";
+import "t2iapi/biceps/localizedtext.proto";
+
+/*
+Represents pm:InstanceIdentifier (defined in IEEE Std 11073-10207-2017).
+*/
+message InstanceIdentifierMsg {
+  CodedValueMsg type = 1;
+  repeated LocalizedTextMsg identifier_name = 2;
+  /*
+  Represents pm:InstanceIdentifier/@Root (defined in IEEE Std 11073-10207-2017).
+  */
+  message RootAttrMsg {
+    string any_u_r_i = 3;
+  }
+  RootAttrMsg root_attr = 4;
```

Comment:
no need for this type indirection, it's a StringValue

Reply:
done

Reply:
Not really though, you made this into a string, not StringValue

Reply:
done